EngineLayer/Ms2ScanWithSpecificMass.cs

```diff
@@ -19,9 +19,12 @@ public Ms2ScanWithSpecificMass(MsDataScan mzLibScan, double precursorMonoisotopi
 
             TheScan = mzLibScan;
 
-            ExperimentalFragments = neutralExperimentalFragments ?? GetNeutralExperimentalFragments(mzLibScan, commonParam);
+            if (commonParam.DissociationType != DissociationType.LowCID)
```

Comment:
I'm confused. Why is this not relevant for lowCID? This populates the experimental fragments and the deconvoluted masses.

Reply:
lowCID doesn't use the deconvoluted MS2 data. the xcorr score uses the peaks in the scan, not deconvoluted masses